plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/package-lock-expected-output.yml

```diff
@@ -1,6 +1,6 @@
 ---
 project:
-  id: "NPM::<REPLACE_PROJECT_NAME>:1.0.0"
+  id: "NPM::npm-package-lock:1.0.0"
```

Comment:
> not from the filename of the definition file.

True, but what's a bit unclear from just looking at this diff is that the callers actually do set `REPLACE_PROJECT_NAME` to something based on the `definitionFile`'s name.

So maybe better say:

> Callers set `REPLACE_PROJECT_NAME` to something based on the `definitionFile`'s name, which is wrong here as the `name` of the identifier in the result comes from the string value of the `name` property within the `package.json`.